### DISCUSSION TITLE
Welcome to Frigate Discussions!

### DISCUSSION THREAD

**Discussion by blakeblackshear** [blakeblackshear/frigate#350] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Buzztiger** [comment]
Finally a discussion thread. Great idea. So I'm Buzz, located in the Netherlands, very much interested in all things home automation and especially AI in combination with home security.

Frigate is watching over our home, been using it now for a couple of month. I'm very impressed with and grateful for the development speed, the effort @blakeblackshear puts into this project and the various contributions from everyone. I would like to contribute to user support and, when I finally find the spare time from work and family, to model customisation.

**Comment by mario-tux** [comment]
Hi, this is Mario from Italy. I use Frigate since 2-3 months. The last beta is covering many of my wishes for a smart and HA-integrated NVR system. Thank you to @blakeblackshear for his efforts in this Open Source project.

**Comment by jamos77** [comment]
Hi,  I'm Jamos from the UK.  I've been using frigate for over a year and it's just gone from strength to strength so great work @blakeblackshear and thank you. Frigate has reduced the number of False positive notifications considerably and with some fine tuning probably entirely compared to my existing NVR solution so I no longer have to monitor my cameras through that now.   I hope to help out supporting other users with config issues and share ideas on how we can use frigate in automations in Home Assistant.

**Comment by bm55** [comment]
Hi All,
This is Brian from NW US. I am not a programmer but very interested in using my IP camera to detect objects to automate things. I have been using frigate for over a year now and love that @blakeblackshear is keep adding amazing functions. Every week or so I read the HA community page to find out if there is something new to learn and post questions about trouble I am having.

**Reply by Ysbrand** [reply]
Hi,
I'm Ysbrand from the Netherlands. I discovered Frigate about a week ago, initially started with the "original" release and jumped to the Beta 3 days ago.
Running Frigate on a PI4 as a Home Assistant Add-on (HASS OS 5.3 booting from SSD)

**Comment by ebendl** [comment]
Hi all,

Eben from South Africa. Been running Home Assistant for about 2.5 years, got wind of Frigate around a year ago. We have a pretty serious crime problem in SA, with thieves quite used to outdoor PIR, electrical fencing, walls, armed response and "normal" alarm systems etc. 

So decided to give frigate a go with the 2 Hikvision IP cameras I already had and an old laptop (USB 2.0 only). Had to ask somebody to buy the Coral in the US and bring it back with him. It worked well, so at some point this year I upgraded to a ThinkCentre M700 i5 with 8GB (and USB3!) and bought 5 more Hikvision cameras. Frigate has been running super solid since and detects people very reliably. 

Haven't had to use it for a real crime situation -- yet -- but feel quite good with it protecting my property and family!

Thanks to blake for an awesome project and all his time and effort he's putting into it!!

**Comment by jlaitinen81** [comment]
Hi! I’m Jani from Finland and extremely interested in home automation.

I learned about Frigate NVR through the Home Assistant FB page a few days ago and am already convinced that I made the right NVR choise. Really happy with what you have accomplished already and looking forward to all the new features.

**Comment by kucau0901** [comment]
hi, Im Yusri From Malaysia. Frigate has been running awesomely protecting my home and my family.  Planning to run frigate in few houses at  my  community and  using telegram group to enhance our security. now running beta ver, tqvm @blakeblackshear

**Comment by zeliant** [comment]
Hi, I'm Terence from Malaysia too. @kucau0901 hi fellow Malaysian

Thank you @blakeblackshear for this wonderful project. Been running Frigate since 0.6.0. Hard to keep up with the speed of your updates with few of my setups already running well. The newly added features are too hard to resist so I'm spending my time moving each setup into 0.8.0 beta.

**Comment by tagir-gadelshin** [comment]
Hi, all!

I'm Tagir from Russia. I work for a US-based company that makes a VMS. (We are becoming top-5 VMS globally). We make our VMS as open for integrations as possible. We also plan to go open-source (but not yet, unfortunately). I won't mention brands for now, so it won't be treated as spam or ad.

One of our users mentioned that he was able to integrate Frigate with our software. I watched videos and went over a description and it looks really interesting. 

@blakeblackshear we can help you publish Frigate in our marketplace as integration for our software -- let me know if you are interested in this.

**Comment by TheSleepySlee** [comment]
Hello, Sleepy from France. thanks @blakeblackshear for this great program. Was wondering if I should switch asking questions here or in HASS forum? I'm not a HASS user.
Using frigaate since couple of months, running on LXC container in proxmox + coral.
Using Blueiris too. 4/5 cams.

**Reply by paularmstrong** [reply]
You're more likely to get good responses here that are easier to follow. The thread on the Home Assistant forum has a lot of noise in it.

**Reply by wpeterw** [reply]
Could you give a few hints on how you've set this up with LXC. Currently running frigate in a VM under Proxmox. Do I install Docker in a Debian LXC ?

**Reply by TheSleepySlee** [reply]
@wpeterw have a look here: https://github.com/blakeblackshear/frigate#virtualization
And yes, docker in LXC.

**Comment by vincent-d-izo** [comment]
hi,
first congratulation for this awesome add-ons
it work straight with a nuc gen 10 and a coral on it. i was looking for a soft like that since coral is available.
do you think it is possible to add a license plate reader to frigate ?

**Reply by TheSleepySlee** [reply]
You could still add it manually using openalpr.
However alpr generally needs dedicated camera, especially at night, to have good results. Plus if you want to use the coral, this would mean trained model for this usage, I have not seen this so far.

**Reply by vincent-d-izo** [reply]
for the camera if IR tuning is available also if the position is aligned with the front or the rear with the moving vehicle you can avoid too much motion blur on the plate.
there is some test with tensorflow and license plate pretrained model. i always think is a huge + to stay outside a cloud system.
i'm not skill enough to dig into that, but when i see frigate,  it's the first time i feel all of that is more accessible for user like me.

**Reply by TheSleepySlee** [reply]
You can run openalpr locally, this is what I am doing. Using CPU.
The thing here I believe would be to use the same coral as frigate, to have better results on trained model, but I don't think it's easily doable.

**Reply by vincent-d-izo** [reply]
ha ok, i didn't know the possibility on local.
yes the dream on a user side, it's to have a just in the config of frigate :  objects : track:  licenseplate :-)

**Comment by matteopizzoli** [comment]
hi, i'm Matteo from Italy, i'm usining Frigate since september 20, and i think it is a wonderful work!
